infertrade/utilities/operations.py

```diff
@@ -413,21 +428,19 @@ def restricted_function(*args, **kwargs) -> pd.DataFrame:
         # Get allocation dataframe from allocation function
         dataframe = allocation_function(*args, **kwargs)
         
-        
-        # Restrictions by limiting allocations between range
-
-        allocation_lower_limit=0
-        allocation_upper_limit=1
-
-        if "allocation_lower_limit" in kwargs:
+        # Restriction by limiting allocations between range
+        if "allocation_lower_limit" in kwargs and "allocation_upper_limit" in kwargs:
+            allocation_upper_limit=allocation_lower_limit=None
             allocation_lower_limit=kwargs.pop("allocation_lower_limit")
```

Comment:
Pop will edit the kwargs (removing the entry), which can cause bugs. Can we just read the value?